### DISCUSSION TITLE
Task Description

### DISCUSSION THREAD

**Discussion by Daek-wml** [Vdauphin/HeartsAndMinds#1619] [Question and answer] [answered]
Hello,

Please guide me on where I can go to edit the descriptions of the tasks? and how can I remove location names from them?

Thanks
![Screenshot 2024-05-28 082911](https://github.com/Vdauphin/HeartsAndMinds/assets/87691381/be840f16-f490-441c-b65d-63373b759fb3)

**Comment by Vdauphin** [comment, marked as answer]
Hello,

Here is the stringtable key: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/task/setDescription.sqf#L89-L97

Here is the string: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/stringtable.xml#L1636-L1653

Cheers
